TestPlans/scratch.xctestplan

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Is there a more descriptive name for this test plan. It sounds like a temporary "I'm just trying stuff" test plan.